Update virtual-machines-windows-ps-sql-keyvault.md

Max-Meng opened the pull request:
The current version missed a few information that might cause confusion in the next steps.

azurecla commented:
Hi __@vivek-dhayalan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@jimdial / @cherylmc - Could you review this proposed update to a load balancer article by @vivek-dhayalan in the public repo?

tysonn commented:
@jimdial /@cherylmc - second notice, please review

jimdial commented:
#sign-off

Good catch, thanks!

Max-Meng committed:
Update virtual-machines-windows-ps-sql-keyvault.md

The current version missed a few information that might cause confusion in the next steps.

GitHubber17 committed:
Update virtual-machines-windows-ps-sql-keyvault.md
